torchtune/utils/metric_logging.py

```diff
@@ -317,3 +317,86 @@ def close(self) -> None:
         if self._writer:
             self._writer.close()
             self._writer = None
+
+
+class CometLogger(MetricLoggerInterface):
+    """Logger for use w/ Comet.ml (https://www.comet.ml/).
```

Comment:
```suggestion
    """Logger for use w/ Comet (https://www.comet.com/site/).
```
We changed our domain some time ago